### ISSUE TITLE
Encryptor provider config for 1.13 use old flag

### ISSUE THREAD

**Issue by pipo02mix** [aquasecurity/kube-bench#262]
I have seen in the 1.13 master config there rule for the encryptor provide config still using `experimental` flag

```yaml
id: 1.1.34
    text: "Ensure that the --experimental-encryption-provider-config argument is
    set as appropriate (Scored)"
    audit: "ps -ef | grep $apiserverbin | grep -v grep"
    tests:
      test_items:
      - flag: "--experimental-encryption-provider-config"
        set: true
``` 

In [Kubernetes docs](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/#configuration-and-determining-whether-encryption-at-rest-is-already-enabled) it says from `1.13` the flag will be `--encryption-provider-config`

I am providing a PR for this

**Comment by ttousai** [comment]
@pipo02mix thank you for catching this and the PR. 

`kube-bench` checks the `experimental-encryption-provider-config` flag to follow the [CIS kubernetes benchmark v1.4.0](https://www.cisecurity.org/benchmark/kubernetes/) recommendations as closely as possible.

In light of this we will raise this as a bug with CIS.
cc @lizrice

**Comment by lizrice** [comment]
This is now fixed through the support of version 1.4.1 of the CIS spec

**Comment by pipo02mix** [comment]
great
